gravitee-plugin-repository/src/main/java/io/gravitee/plugin/repository/internal/RepositoryPluginHandler.java

```diff
@@ -211,7 +211,16 @@ private void checkRepositoryConfig(final Scope scope, final boolean failOnMissin
     }
 
     private String getRepositoryType(Scope scope) {
-        return environment.getProperty(scope.getName() + ".type");
+        String repositoryType = environment.getProperty("repositories." + scope.getName() + ".type");
+        if (repositoryType != null) {
+            return repositoryType;
+        } else {
+            LOGGER.warn(
+                "Repository of scope {} is loaded from old structure. Please move repositories settings under 'repositories' section.",
+                scope
+            );
```

Comment:
I wondering if we have to add this warn only if the type is found with the legacy key :thinking: 
Because if the type is **not found** line 222, the warn make no sens IMO. It is useful only when line 222 return a non null value.

wdyt ?

Reply:
valid point :). resolved.